ludwig/backend/ray.py

```diff
@@ -147,8 +147,9 @@ def _create_pandas_engine(**kwargs):
 
 
 def _get_df_engine(processor):
-    logger.info(f"Ray processor params: {processor}")
-    if processor is None:
+    if processor:
```

Comment:
Can clean this up:

```
if processor is None:
    return _create_dask_engine()

logger.info(f"Ray processor params: {processor}")
processor_kwargs = processor.copy()
...
```

That way we don't need to keep extra if-else statements (too many if-elses are generally considered an anti-pattern).